x/group/msgs_test.go

```diff
@@ -1,7 +1,7 @@
 package group_test
 
 import (
-	"fmt"
+	fmt "fmt"
```

Comment:
Why?

Reply:
Truly don't know

Reply:
This is super weird I think I'm going to revert that file to its state from main and make sure that no tooling did that.